.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,10 @@
 ## Product Description
-<!-- For non-invisible changes, describe user-facing effects. -->
+<!-- For non-invisible changes, describe user-facing effects and include screenshots. -->
```

Comment:
nit: where reasonable?

Just rethinking if someone would feel the need to or would be asked to put screenshots for changes that don't need one.